_targets.R

```diff
@@ -21,12 +24,18 @@ list(
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
   tar_map(
-    values = tibble(state_abb = states),
+    tibble(state_abb = states) %>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
     tar_target(nwis_inventory, dplyr::filter(oldest_active_sites,
                                              state_cd == state_abb)),
-    tar_target(data, get_site_data(nwis_inventory, state_abb, parameter))
+    tar_target(data, get_site_data(nwis_inventory, state_abb, parameter)),
     # Insert step for tallying data here
+    tar_target(tally, tally_site_obs(data)),
     # Insert step for plotting data here
+    tar_target(timeseries_png, plot_site_data(state_plot_files,
+                                              data,
+                                              parameter)),
```

Comment:
Note that `plot_site_data()` produces a _**file**_ and this `tar_target()` call will need an additional argument to handle files.